torch/_inductor/utils.py

```diff
@@ -680,6 +680,15 @@ def patched_compile_to_module(self):
     return result, source_codes
 
 
+def print2(*args):
```

Comment:
Why is this called print2?

Reply:
I found it useful to have a `print` utility that only printed out on the first rank for debugging purposes. Will probably keep in the PR but rename.